### PR TITLE
Website Italian translation

### PR DESCRIPTION
**Pull request by Franciman**
Hi, here's an Italian translation for the website.
Just one question for Hungarians: Is `Olasz` the correct Hungarian translation of `Italian` (as in the language)?
(CC @notramo )

### CODE REVIEW

**Comment by lenormf** [comment]
Could you rebase your branch on `master`, and squash all the commits?

**Comment by notramo** [comment]
@Franciman yes.

File: index.html [hunk 1]
```diff
@@ -132,7 +134,6 @@ <h2 tid="menu:features">Features</h2>
                                 <i class="fa fa-fw fa-object-group"></i>
                                 <span tid="feature:multipleSelections">Multiple selections</span>
                             </div>
-
```
Comment (lenormf) [inline index.html]: Could you remove this hunk?

Comment (Franciman) [reply index.html]: Done.

**Pull request merged**